src/components/ProgressBar/ProgressBar.unit.spec.jsx

```diff
@@ -12,5 +13,91 @@ describe('<ProgressBar/>', () => {
     tree.unmount();
   });
 
-  it('should pass aXe check', () => axeCheck(<ProgressBar percent={35} label="aria label here"/>));
+  it('should pass aXe check', () =>
+    axeCheck(<ProgressBar percent={35} label="aria label here" />));
+
+  describe('analytics event', function () {
+    it('should be triggered when a ProgressBar is mounted at 0%', () => {
```

Comment:
Is it ok that the event will have the `change` action?

Reply:
I was actually thinking about this too.   On the GTM side, we don't make a distinction between mounting at 0% and a change back to 0%.  I could change to `load` maybe, but it would probably still enter GA as a "change".

Reply:
Sorry - I should have mentioned that I would prefer to keep this as `change` if possible.    Maybe consider it the `change` from _undefined_ to _0_?    

¯\\\_(ツ)\_/¯